mariadb.spec

```diff
@@ -118,12 +119,12 @@
 
 # Make long macros shorter
 %global sameevr   %{epoch}:%{version}-%{release}
-%global compatver 10.1
-%global bugfixver 18
+%global compatver 10.0
+%global bugfixver 28
```

Comment:
woops... I just created https://github.com/rdo-common/mariadb/pull/8 to fix that.